Formula/tracker.rb

```diff
@@ -0,0 +1,46 @@
+class Tracker < Formula
+  desc "Library and daemon that is an efficient search engine and triplestore"
+  homepage "https://gitlab.gnome.org/GNOME/tracker"
+  url "https://download.gnome.org/sources/tracker/3.1/tracker-3.1.2.tar.xz"
+  sha256 "da368962665d587bb2e4f164d75919a81dacb35c7d4cfae6f93a94c60f60ec8f"
+  license all_of: ["LGPL-2.1-or-later", "GPL-2.0-or-later"]
+
+  # Tracker doesn't follow GNOME's "even-numbered minor is stable" version scheme.
+  livecheck do
+    url :stable
+    regex(/tracker-(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  depends_on "gobject-introspection" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "vala" => :build
+  depends_on "dbus"
+  depends_on "json-glib"
+  depends_on "libsoup"
+  uses_from_macos "icu4c"
+
+  def install
+    args = std_meson_args + %w[
+      -Dman=false
+      -Ddocs=false
+      -Dsystemd_user_services=false
+    ]
+
+    ENV["DESTDIR"] = "/"
+    mkdir "build" do
+      system "meson", *args, ".."
+      system "ninja", "-v"
+      system "ninja", "install", "-v"
+    end
+  end
+
+  def post_install
+    system "#{Formula["glib"].opt_bin}/glib-compile-schemas", "#{HOMEBREW_PREFIX}/share/glib-2.0/schemas"
+  end
+
+  test do
+    system "#{bin}/tracker3", "--help"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Ideally I would just run the meson tests, but I don't see other packages doing that (and I'm not sure if I can access the build folder during tests or would have to re-build). I suppose I could re-purpose one of Tracker's example otherwise.

Reply:
I've added a proper test. Thanks for the quick review.